webui/webui/components/chat.py

```diff
@@ -14,23 +16,23 @@ def message(qa: QA) -> rx.Component:
     Returns:
         A component displaying the question/answer pair.
     """
-    return rx.chakra.box(
-        rx.chakra.box(
+    return rx.box(
+        rx.box(
```

Comment:
Should we change the inner components to `rx.text`

Reply:
fixed